test/Well.RemoveLiquidityImbalanced.t.sol

```diff
@@ -30,23 +32,27 @@ contract WellRemoveLiquidityImbalancedTest is TestHelper {
 
     /// @dev Base case
     function test_removeLiquidityImbalanced() public prank(user) {
+        uint initialLpAmount = 2000 * 1e27;
```

Comment:
Is this: `initialLpAmount = addedLiquidity + initialLiquidity`? Don't necessarily have to make it a var